### PR TITLE
feat: add ability to have function as linkTo parameters

### PR DESCRIPTION
**Pull request by mthuret**
Link to #22

Allow the ability to filter to redirect to the right story. 

This feature was existing but seems to have disappeared.

### CODE REVIEW

**Comment by mthuret** [comment]
what do you think of this PR @mnmtanish ?

File: src/preview.js [hunk 1]
```diff
@@ -2,8 +2,12 @@ import addons from '@kadira/storybook-addons';
...
 
 export function linkTo(kind, story) {
-  return function () {
+  return function (...args) {
+    const resolvedKind = typeof kind === 'function' ? kind(...args) : kind;
+    const resolvedStory = typeof story === 'function' ? story(...args) : story;
+    
     const channel = addons.getChannel();
-    channel.emit(EVENT_ID, {kind, story});
+    console.log(resolvedKind)
```
Comment (thani-sh) [inline src/preview.js]: Please remove this line

**Review by thani-sh — CHANGES_REQUESTED** [review]
Cool feature. I'll merge this once the changes are done.

**Comment by mthuret** [comment]
Done :)

**Comment by mthuret** [comment]
@mnmtanish any news?

**Pull request merged**